hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -656,6 +657,11 @@ public void initialize(URI name, Configuration originalConf)
       // verify there's no S3Guard in the store config.
       checkNoS3Guard(this.getUri(), getConf());
 
+      // performance creation flag for code which wants performance
```

Comment:
can we move comment here to where this property MACRO is defined? easy to find there.